pom.xml

```diff
@@ -1,23 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.danielfrak.code.keycloak.providers</groupId>
     <artifactId>keycloak-rest-provider</artifactId>
     <version>1.0-SNAPSHOT</version>
+
+    <scm>
+        <developerConnection>scm:git:https://github.com/${git.repository}.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <distributionManagement>
+        <repository>
+            <id>github</id>
+            <name>GitHub</name>
+            <url>https://maven.pkg.github.com/${git.repository}</url>
+        </repository>
+    </distributionManagement>
+
     <properties>
         <java.version>11</java.version>
-        <keycloak.version>12.0.0</keycloak.version>
-        <resteasy.version>4.4.1.Final</resteasy.version>
+        <keycloak.version>13.0.1</keycloak.version>
+        <resteasy.version>3.15.1.Final</resteasy.version>
```

Comment:
The version mentioned in this PR is lower than in the project itself. Is that on purpose?

Reply:
Looks like it since you've created a separate commit for this :+1:

Reply:
Can you highlight where do you see the differenc? In this pull request I wrote "Set resteasy-client version to 3.15.1.Final because keycloak use this version" and I see the exact version in the pom.

Reply:
Indeed, the change was on purpose. When I wrote "project" I referred to this keycloak-user-migration project and not keycloak iself. All is well in other words!